### ISSUE TITLE
Bug: load game selector off screen at the top

### ISSUE THREAD

**Issue by walkawayy** [rr-/Tomb1Main#308]
If you load a game in the last few slots, but your most recent save is at the top, the most recent save is selected at the top of the list off screen.

**Comment by rr-** [comment]
Fixed in https://github.com/rr-/Tomb1Main/commit/ce4b2cb6be4ea4bfbb9322a7abe5026dd6f6911f

**Comment by walkawayy** [comment]
Ok I messed up this PR. Noticed it while testing something else.

The bug was loading an old save at the bottom of the passport and then going back to load game where a new save at the top of the list is highlighted off scree. See:
![screenshot0005](https://user-images.githubusercontent.com/81546780/148669889-f6d4407e-c2e2-409f-903f-7c85a1279e29.jpg)

Now the bad fix I pushed could do this:
![screenshot0006](https://user-images.githubusercontent.com/81546780/148669931-e1d6e62a-fe6a-4aa4-b0e0-8ed5e05e71eb.jpg)



Now both these cases work with what I pushed.
![screenshot0008](https://user-images.githubusercontent.com/81546780/148670009-a4c6b17c-182e-453c-b799-2fd5a5c03451.jpg)

![screenshot0008](https://user-images.githubusercontent.com/81546780/148670014-20c320ec-6641-4873-aae4-7aca976afad9.jpg)
